### DISCUSSION TITLE
Toucan Samples and Mustache.js

### DISCUSSION THREAD

**Discussion by nonfungibletunji** [toucansites/toucan#50] [Q&A] [answered]
``` <a href="{{url}}"{{#class}} class="{{.}}"{{/class}}>{{label}}</a> ```

what does the mustache.js curly brackets like {{#class}} and {{.}} produce

**Comment by viaszkadi** [comment, marked as answer]
Mustache is a logic-less templating language used for rendering data by replacing placeholders with data values. The `{{ }}` syntax in Mustache denotes placeholders where variables or expressions are inserted when the template is rendered. [This manual](https://mustache.github.io/mustache.5.html) explains the basics well.
